src/sentry/conf/server.py

```diff
@@ -1353,8 +1353,10 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
     "organizations:create": True,
     # Enable usage of customer domains on the frontend
     "organizations:customer-domains": False,
-    # Allow disabling integrations when broken is detected
-    "organizations:slack-disable-on-broken": False,
+    # Allow disabling github integrations when broken is detected
+    "organizations:github-disable-on-broken": False,
+    # Allow disabling slack integrations when fatal broken is detected
+    "organizations:slack-disable-fatal-on-broken": False,
```

Comment:
Isn't the slack disable fatal one from a different PR? Might be a weird rebase thing going on here.